installer/scripts/omsadmin.sh

```diff
@@ -677,13 +739,20 @@ list_workspaces()
             found_ws=1
             show_workspace_status ${ws_conf_dir} ${ws_id} 1
         fi
+
+        # check scom workspace
+        list_scom_workspace
+        found_ws=$?
     else
         # no default conf folder, check all the potential workspace folders
         for ws_id in `ls -1 $ETC_DIR | grep -E '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'`
         do
             found_ws=1
             show_workspace_status $ETC_DIR/${ws_id}/conf ${ws_id} 0
         done
+        # check scom workspace
+        list_scom_workspace
+        found_ws=$?
```

Comment:
Seem to me like you use this block in all cases: if there is a symlink at conf/, if conf/ exists but isn't a symlink, and the else case. Why not just move this whole block to right before `if [ $found_ws -eq 0 ]; then` ?